Firestore/Protos/README.md

```diff
@@ -2,22 +2,16 @@
 
 First, make sure you have necessary prereqs for building:
 ```
-brew install automake libtool protobuf
+brew install automake libtool protobuf golang
```

Comment:
Is anything but golang required anymore?

Reply:
Hm. Not sure. When I attempted this using a linux container (docker), I think the only extra stuff I needed was go and python-six. But, I think I installed the basic dev tools first, which would've included gcc, make, and I believe, the autotools. So the only thing I'm fairly confident we don't need is protobuf, since the cmake build pulls that in. And it's possible we need some additional python libraries?

We should give this a run through on a fresh mac install.